changelog.txt

```diff
@@ -1,3 +1,17 @@
+4.14.0
+============
+New Features: 
+
+**MSAL no longer calls the OIDC endpoint, as it can infer the authorization and token endpoints based on the authority URL. This will speed up token acquisition, especially for multi-tenant applications, as fewer network calls will be made. For details see [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1750)
+
+**Client throttling extended support** MSAL will throttle all /token calls during an event in which the server sends a Retry-After header, thus ensuring the Retry-After instruction is observed. MSAL will also throttle server requests that result in `MsalUiRequiredException` being thrown, for example when the user is required to perform MFA but the app keeps trying to acquire a token silently. For details see [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1624)
+
+Bug Fixes:
+
+**MSAL .NET now respects the ValidateAuthority=false flag. See [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1791)
+
+**When the Andrid borker is configured but it is not installed, MSAL should revert to using its own cache to try to perform the AcquireTokenSilent call. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1809)
```

Comment:
```suggestion
**When the Android borker is configured but it is not installed, MSAL should revert to using its own cache to try to perform the AcquireTokenSilent call. [Issue for details](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1809)
```

Reply:
Also, what is a "borker" (broker?)

Reply:
This is the "Authenticator" and  "Company Portal" apps on iOS and Android. If you use those to intermediate the logging in process, the user does not need to re-enter their credentials.